CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 ------------
 
 * Your contribution here.
+* Add ability to customize tracker class for each trackable; multiple trackers across the app are now possible - [@johnnyshields](https://github.com/johnnyshields).
```

Comment:
You're not `@johnnyshields` AFAIK :) Change this to have your name and link to the issue as below, please.